qwiic_i2c/__init__.py

```diff
@@ -63,11 +63,24 @@
 #
 #-----------------------------------------------------------------------------
 # Drivers and driver baseclass
-from .i2c_driver 	import I2CDriver
-from .linux_i2c 	import LinuxI2C
-from .circuitpy_i2c import CircuitPythonI2C
+from .i2c_driver import I2CDriver
 
-_drivers = [LinuxI2C, CircuitPythonI2C]
+# All supported platform module and class names
+sub_modules = ["linux_i2c", "circuitpy_i2c", "micropython_i2c"]
+class_names = ["LinuxI2C", "CircuitPythonI2C", "MicroPythonI2C"]
+
```

Comment:
One thing that you could do here - use a dict - where you have the sub-module the key and the contents be the class name .Something like:
```pythong
canidate_drivers = { "linux_i2c" : "LinuxI2C", 
                                   "circuitpy_i2c" : "CircuitPythonI2C",
                                  "micropython_i2c" : "MicroPythonI2C"}
  for mod, obj  in canidate_drivers.items():
      try:
          sub_module = __import__("qwiic_i2c."+mod)
          _drivers.append(getattr(sub_module, obj)
     except:
           pass

```
Or something like that -